### ISSUE TITLE
Shouldn't `FileContract.Filesize` be `FileContract.FileSize` for consistency?

### ISSUE THREAD

**Issue by mike76-dev** [SiaFoundation/core#90]
https://github.com/SiaFoundation/core/blob/bc0a972fae409421a298bd37a729164a30caf184/types/types.go#L224

**Comment by lukechampine** [comment]
Ha, yeah, it's a personal style quirk of mine. Is it one word ("filesize") or two ("file size")? I've always thought of it as one word. Note that this isn't the only change like this: e.g. `Transaction.TransactionSignatures` is now `Transaction.Signatures`.

Frankly, calling it a "file" size in a "file" contract hasn't been accurate since, like, 2015, when we added the ability to store multiple files within one contract. "Storage contract" makes more sense. And yet, I feel a strange attachment to "file contract." We might just be stuck with it. 😄

**Comment by mike76-dev** [comment]
I only raised this issue, because in `siad` the corresponding field is called `FileSize`. I was writing a converter and stumbled upon this.



On Monday, February 20, 2023, 6:04 PM, Luke Champine ***@***.***> wrote:




Ha, yeah, it's a personal style quirk of mine. Is it one word ("filesize") or two ("file size")? I've always thought of it as one word. Note that this isn't the only change like this: e.g. Transaction.TransactionSignatures is now Transaction.Signatures.

Frankly, calling it a "file" size in a "file" contract hasn't been accurate since, like, 2015, when we added the ability to store multiple files within one contract. "Storage contract" makes more sense. And yet, I feel a strange attachment to "file contract." We might just be stuck with it. 😄
